script/Ciel/Day 11/11_01_CIEL11_2_1.txt

```diff
@@ -392,7 +392,7 @@ You got lucky and reincarnated into a brat with some kinda vampire-like ability,
 [sha:bad1e43bcd59a2ef393587560673c7b81fb71718]{
 -- Page 117, Offset 29400.
 -- 悲惨な話だがね。この街には、一心同体の人でなしが二人もいた事になる」
-Ain't it fuckin' tragic.
+Ain't it fuckin' tragic?
```

Comment:
Hmmmm I don't like the tag question ? here but I also don't like it in general so hard to say if that's signal or not.

Reply:
I'll just revert it then.